### ISSUE TITLE
[undefined] The NVDA doesn't read "Facebook/ Twitter/ LinkedIn/ Instagram " icons name when user try to mouse hover on them

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3045]
**Steps to Reproduce**
undefined

**Expected Result**
undefined

**Actual Result**
undefined

**Device**: undefined
**Operating System**: undefined
**Browser**: undefined

**WCAG Category**: undefined
**WCAG Level**: undefined
**WCAG Success Criterion**: undefined
**Screen Reader Used**: undefined
**Accessibility Audit Tool used**: undefined

**Attachment**
undefined
